grammar.js

```diff
@@ -1,4 +1,5 @@
 const PREC = {
+  await:15,
```

Comment:
Also, I think you can use `PREC.field` (since `x.await` probably has the same precedence as `x.y`) rather than adding a new precedence constant.